tailwind.config.js

```diff
@@ -40,15 +40,19 @@ module.exports = {
       spacing: {
         'neg-sm': '-21px',
         'neg-md': '-22px',
+        '2xs': '-12px',
         '3xs': '7.5px',
         xxs: '8px',
         sm: '10px',
         md: '18px',
+        '30': '30px',
         'md-40': '40px',
+        '1xl': '210px',
         '2xl': '300px',
         '3xl': '380px',
         '38': '412px',
-        '39': '623px'
+        '39': '623px',
+        '75': '1200px'
```

Comment:
https://tailwindcss.com/docs/width#app
https://tailwindcss.com/docs/max-width#app

Please, check the default sized. 
And please, round minor size difference, e.g. 
623px / 4 = 155.75 ~=160 = 40rem (I think that acceptable in this case use 42rem - .max-w-2xl)